src/gluonts/ev/__init__.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from .metrics import (
+    Metric,
+    mean_absolute_label,
+    sum_absolute_label,
+    SumAbsoluteError,
+    MSE,
+    SumQuantileLoss,
+    Coverage,
+    MAPE,
+    SMAPE,
+    MSIS,
+    MASE,
+    ND,
+    RMSE,
+    NRMSE,
+    WeightedSumQuantileLoss,
+)
+from .aggregations import Aggregation, Sum, Mean
+from .evaluator import Evaluator, DirectEvaluator, DerivedEvaluator
+
+__all__ = [
```

Comment:
Should we add `seasonal_error` from `ts_stats` here as well?

Reply:
Also, should we expose all `stats`?

Reply:
Sure. 👍